src/content/en/updates/2020/08/devtools.md

```diff
@@ -0,0 +1,261 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: New Media panel, Issues tab updates, emulate missing local fonts, inactive users and prefers-reduced-data.
+
+{# wf_updated_on: 2020-08-20 #}
+{# wf_published_on: 2020-08-20 #}
+{# wf_tags: chrome86, devtools, devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: New Media panel, Issues tab updates, emulate missing local fonts, inactive users and prefers-reduced-data. #}
+{# wf_blink_components: Platform>DevTools #}
+
+# What's New In DevTools (Chrome 86) {: .page-title }
+
+{% include "web/_shared/contributors/jecelynyeen.html" %}
+
+
+## New Media panel {: #media-panel }
+
+DevTools now displays media players information in the [Media panel](/web/tools/chrome-devtools/media). 
+
+![New Media panel](/web/updates/images/2020/08/media-panel.png)
+
+Prior to the new media panel in DevTools, logging and debug information about video players could be found in [chrome://media-internals](chrome://media-internals).
+
+The new Media panel provides an easier way to view events, logs, properties, and a timeline of frame decodes in the same browser tab as the video player itself. You can live view and inspect on potential issues quicker (e.g. why dropped frames are occurring, why JavaScript is interacting with the player in an unexpected way).
+
+For engineers who are constantly testing video playback quality, working on a JavaScript front-end to a media player, or working on re-encoding tools, this new Media panel could greatly increase your productivity.
+
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/bf98fea787a8c2db4e0763ae2095faf8112458eb #}
+
+Chromium issues: [1018414](https://crbug.com/1018414)
+
+
+## Issues tab updates {: #issues-tab }
+
+The Issues warning bar on the Console panel is now replaced with a regular message. 
+
+![Issues in console message](/web/updates/images/2020/08/issue-console-msg.png)
+
+Third party cookie issues are now hidden by default in the Issues tab. Tick the newly added “Include third-party cookie issues” checkbox to view them.
```

Comment:
```suggestion
Third-party cookie issues are now hidden by default in the Issues tab. Enable the new **Include third-party cookie issues** checkbox to view them.
```
I find "enable" is a little more precise of a term than "tick"

Bolding UI elements per style guide: https://developers.google.com/style/ui-elements#formatting